<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KReconciliationReportDlgDecl</class>
 <widget class="QDialog" name="KReconciliationReportDlgDecl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconciliation report</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="KTabWidget" name="m_tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="m_summaryTab">
      <attribute name="title">
       <string>Summar&amp;y</string>
      </attribute>
      <layout class="QVBoxLayout" name="m_summaryLayout"/>
     </widget>
     <widget class="QWidget" name="m_detailsTab">
      <attribute name="title">
       <string>&amp;Details</string>
      </attribute>
      <layout class="QVBoxLayout" name="m_detailsLayout"/>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_okButton">
       <property name="text">
        <string>O&amp;K</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_printButton">
       <property name="text">
        <string>Pri&amp;nt</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">ktabwidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_okButton</sender>
   <signal>clicked()</signal>
   <receiver>KReconciliationReportDlgDecl</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
